package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Lab03--Async",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
delate lin 7 and replace with:
``` 
    "lint": "eslint **/.js",
    "test": "jest --coverage --verbose",
    "test-watch": "jest --watchAll --coverage --verbose"```

That should help when it comes to testing.

Reply:
Ok I will add